cirq-core/cirq/protocols/json_test_data/spec.py

```diff
@@ -166,10 +166,10 @@
         'ParamMappingType',
         # utility:
         'CliffordSimulator',
-        'NoiseModelFromNoiseProperties',
         'Simulator',
         'StabilizerSampler',
         'DEFAULT_RESOLVERS',
+        'NoiseModelFromNoiseProperties',
```

Comment:
The `'NoiseModelFromNoiseProperties'` should be removed from the `should_not_be_serialized` list if this PR can support serialization of `NoiseModelFromNoiseProperties`.  If that is too complicated, it is OK to limit the scope here to other NoiseModel classes only and leave NoiseModelFromNoiseProperties for later.